Week03/lesson05/classwork/week3-relaxr-part2/css/style.css

```diff
@@ -0,0 +1,241 @@
+/*
+Project Name: Relaxr Blog Homework Lesson 5
+Client: Relaxr
+Author: Christophe Brumby
+Developer @GA in London
+*/
+
+
+/******************************************
+/* SETUP
+/*******************************************/
+
+/* Box Model Hack */
+* {
+ box-sizing: border-box; /* IE */
+}
+
+/* Clear fix hack */
+.clearfix:after {
+  content: ".";
+  display: block;
+  clear: both;
+  visibility: hidden;
+  line-height: 0;
+  height: 0;
+}
+
+.clear {
+ clear: both;
+}
+
+.alignright {
+  float: left;
+  padding: 0 0 10px 0; /* note the padding around a right floated image */
+  width: 50%;
+  text-align: right;
+}
+
+.alignleft {
+  float: right;
+  padding: 0 0 10px 20px; /* note the padding around a left floated image */
+  width: 50%;
+  text-align: left;
+}
+
+/******************************************
+/* BASE STYLES
+/*******************************************/
+
+body {
+  color: #000;
+  font-size: 12px;
+  line-height: 1.4;
+  font-family: Open Sans, Helvetica, Arial, sans-serif;
+}
+
+
+/******************************************
+/* LAYOUT
+/*******************************************/
+
+/* Center the container */
+#container {
+  width: 1100px;
+  margin: auto;
+  background-color: #f0efef;
```

Comment:
Like I say below, I definitely confused you guys with this assignment's starter code. To clarify, you generally want to let `background-color`s occupy the full width of the screen and then use CSS like what your have for `#container` here (which limits the width and centers the div on the page) to wrap around just the content.

So in order to get this effect:

![image](https://cloud.githubusercontent.com/assets/5670279/17464594/61e2f7a4-5cda-11e6-87d7-2f402e9bbe05.png)

... I moved the `background-color` from the line above (66) to the `<body>` and then took the `id="container"` off of the full-page wrapper div and put it on the same div you applied the `clearfix` class on (since that is the parent div that directly wraps around the elements containing the content)